### ISSUE TITLE
Exception: Unable to build code model

### ISSUE THREAD

**Issue by skoniam** [codecadwallader/codemaid#185]
Hi,
Get exception when database project is opening
```
[CodeMaid Handled Exception 07:49:07 AM] Unable to build code model for 'ViewDataWindow_MSSQL__/_localdb__ProjectsV12/ZISAXSONE/True/SqlTable/ZIS.PlikiDoNagraniaLog.sql': System.NullReferenceException: Object reference not set to an instance of an object.
   at SteveCadwallader.CodeMaid.Helpers.TextDocumentHelper.FindMatches(TextDocument textDocument, String patternString)
   at SteveCadwallader.CodeMaid.Model.CodeModelHelper.RetrieveCodeRegions(TextDocument textDocument)
   at SteveCadwallader.CodeMaid.Model.CodeModelBuilder.RetrieveAllCodeItems(Document document)
   at SteveCadwallader.CodeMaid.Model.CodeModelManager.BuildCodeItems(CodeModel codeModel)
```

**Comment by codecadwallader** [comment]
Since this is a SQL file, it is expected that it won't be able to parse the code model.  Are you seeing any impact other than the warning message in the output window?

**Comment by skoniam** [comment]
Yes, I know that, but in my opinion I shouldn't see this message.
No, CodeMain working excellent.

**Comment by codecadwallader** [comment]
I don't disagree about the warning message, but I wanted to confirm you weren't seeing a bigger issue (e.g. a warning dialog).

I haven't reproduced this with a stock SQL Server project.  Some details that would help:

- Visual Studio version
- CodeMaid version
- Code excerpt/file (if applicable)
- Was this working as expected with a different version?
- If you refresh the Spade tool window, does the exception occur again?  Or does it only happen on solution open?
- Does it happen with all SQL files, or only specific ones?

**Comment by skoniam** [comment]
I see this exception from time to time. I try to observe when it happens.

**Comment by codecadwallader** [comment]
Based on the discussion in #192 we will plan on making these warnings quieter, so they don't cause false alarms.

**Comment by codecadwallader** [comment]
Now available in the v0.9.0.1 release https://github.com/codecadwallader/codemaid/releases/tag/v0.9.0.1-beta
